R/plotCCA.R

```diff
@@ -228,7 +228,7 @@ setMethod("plotRDA", signature = c(object = "SingleCellExperiment"),
         if ( !is.numeric(ellipse.linewidth) && ellipse.linewidth > 0 ) {
             stop("'ellipse.linewidth' must be a positive number.", call. = FALSE)
         }
-        if( confidence.level < 0 || confidence.level > 1 ){
+        if( is.numeric(confidence.level) < 0 || is.numeric(confidence.level) > 1 ){
```

Comment:
This is not correct since is.numeric() outputs logical.

--> !(is.numeric(confidence.interval) && confidence.interval > 0 && confidence.interval < 1)

Reply:
Yes sorry

Reply:
confidence.level = TRUE still passes through without error. Also I noticed that other input check are also incorrect. There are multiple ways to think about this if sentence but I think the most simplest is to think first the supported values and then just take negation -->

!(is.numeric(confidence.interval) && confidence.interval > 0 && confidence.interval < 1)

... which means that the value must be numeric and between 0 and 1. If it is not, then it give TRUE.

--> Can you fix these input checks from this function so that they follow this?